src/game/clock.c

```diff
@@ -5,6 +5,13 @@
 #include <stdio.h>
 #include <time.h>
 
+static bool m_IsTurbo = false;
+
+void Clock_SetTurbo()
+{
+    m_IsTurbo ^= true;
```

Comment:
- Please call it `ToggleTurbo`
- Missing `(void)`
- This is a bool. Please use `m_IsTurbo = !m_IsTurbo`. Has no effect but not everyone knows what xor is